web/packages/teleport/src/services/auth/auth.ts

```diff
@@ -282,6 +282,8 @@ const auth = {
     mfaType?: DeviceType,
     totpCode?: string
   ): Promise<MfaChallengeResponse> {
+    if (!challenge) return;
```

Comment:
We return undefined here, but null on line 315. Can we be more consistent and explicit about the behavior?